src/test/ui/aligned_enum_cast.rs

```diff
@@ -11,5 +11,13 @@ enum Aligned {
 fn main() {
     let aligned = Aligned::Zero;
     let fo = aligned as u8;
-    println!("foo {}",fo);
+    println!("foo {}", fo);
+    println!("{}", tou8(Aligned::Zero));
```

Comment:
should this be an assertion instead of a print? Otherwise we're only testing that this code runs, not that it does the right thing

Reply:
I have added assertions but kept the prints, just in case they were relevant to trigger the ICE.